GiTinder/appsettings.json

```diff
@@ -5,7 +5,11 @@
         }
     },
     "AllowedHosts": "*",
-  "ConnectionStrings": {
-    "GiTinderContextMSSqlDb": "Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=novelgitinder;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
-  }
+    "ConnectionStrings": {
+        "GiTinderContextMSSqlDb": "Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=novelgitinder;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
+    }
 }
+//Local:
+    //Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=novelgitinder;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False
+//Remote:
```

Comment:
@Riceqrisp we shouldnt put the remote version of this string anywhere near github